config/domain_variations/BananaCollectors_test.yaml

```diff
@@ -0,0 +1,9 @@
+resampling-interval: 5000
```

Comment:
Why do we need one file per environment? If the goal is to show the functionality, one (maybe two) examples should be enough.
If the goal is to show something else, what is it?

Reply:
The goal is to enable researchers who are looking to reproduce the results in our JAIR paper, or compare to their own method/algorithm.